### ISSUE TITLE
Error when running 'IBMQ device status'

### ISSUE THREAD

**Issue by malmaud** [Qiskit/qiskit-vscode#81]
### Informations

-   **VSCode version**:  1.32.2
-   **Qiskit [Terra / Aqua / other] version**: 
* qiskit 0.8.0
* terra 0.7.1
* aqua 0.4.1
-   **Qiskit VSCode Extension version**: 0.5.3
-   **Operating system**: OS X 10.14.3

### What is the current behavior?

When I click the IBMQ device status button, after successfully following the prompts to input my API key, I see the below error. Reloading the extension doesn't help.

```
Traceback (most recent call last): File "/Users/malmaud/.vscode/extensions/qiskit.qiskit-vscode-0.5.3/resources/qiskitScripts/listRemoteBackends.py", line 16, in <module> from qiskitTools import QiskitTools File "/Users/malmaud/.vscode/extensions/qiskit.qiskit-vscode-0.5.3/resources/qiskitScripts/qiskitTools.py", line 8, in <module> from IBMQuantumExperience import IBMQuantumExperience ModuleNotFoundError: No module named 'IBMQuantumExperience'
```

### Steps to reproduce the problem

Start with a fresh Anaconda distribution and point VSCode to it. Install the extension and click 'yes' at all prompts shown in VSCode to install qiskit packages.

**Comment by cbjuan** [comment]
Thanks for detecting it, @malmaud. We are fixing it in the PR #82

**Comment by cbjuan** [comment]
Released new version [v0.5.4](https://github.com/Qiskit/qiskit-vscode/releases/tag/v0.5.4). Please update your extension in VSCode, it should solve the issue and let you use the extension as expected.
